core-primitives/sgx/crypto/src/lib.rs

```diff
@@ -16,14 +16,15 @@ pub mod sgx_reexport_prelude {
 	pub use serde_sgx as serde;
 }
 
+pub mod error;
+pub mod traits;
+
 #[cfg(feature = "sgx")]
 pub mod aes;
 #[cfg(feature = "sgx")]
 pub mod ed25519;
-pub mod error;
 #[cfg(feature = "sgx")]
 pub mod rsa3072;
-pub mod traits;
```

Comment:
Just to make it more obvious and readable which modules are available in `std` as well